.gitignore

```diff
@@ -112,3 +112,4 @@ logo.png
 /src/DynamoCoreWpf/package-lock.json
 /src/DynamoCoreWpf/package.json
 /src/DynamoCoreWpf/.npm-cache
+/src/DynamoSandbox/node_modules
```

Comment:
thank you!

Reply:
Wait why is this needed? I thought we moved Splash Screen to `DynamoCoreWpf` so Sandbox project should no longer download npm packages..

Reply:
I was not supposed to submit this but I have had issues with this in the past. I will remove it and let you know if I still need it.